lib/Feature/FeatureModel.cpp

```diff
@@ -78,19 +78,44 @@ bool isSimpleMutex(const Feature *A, const Feature *B) {
       });
 }
 
+/// Collect mutual exclusive feature constraints for clean up.
+llvm::SmallSet<const Constraint *, 3>
+cleanUpMutex(const Feature *A, const llvm::SmallSet<Feature *, 3> &Xor) {
```

Comment:
We could come up with a better name :laughing: My first thought was "why is he using a mutex here"
Maybe `cleanUpMutualExclusiveConstraints` ?